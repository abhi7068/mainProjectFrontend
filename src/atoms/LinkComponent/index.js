import React from 'react';
import PropTypes from 'prop-types';
function LinkComponent({ children }) {
  return (
    <div
      style={{
        fontFamily: 'Roboto, sans-serif',
        fontWeight: '500',
        fontSize: '14px',
        margin: '0',
        whiteSpace: 'nowrap',
      }}
    >
      {children}
    </div>
  );
}
LinkComponent.propTypes = {
  children: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
};

export default LinkComponent;
