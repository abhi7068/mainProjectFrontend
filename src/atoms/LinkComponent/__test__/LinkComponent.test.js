import React from 'react';

import LinkComponent from '../index';
import { render } from '@testing-library/react';

it('it renders', () => {
  render(<LinkComponent />);
});
it('should take a snapshot', () => {
  const { asFragment } = render(<LinkComponent />);
  expect(asFragment(render(<LinkComponent />))).toMatchSnapshot();
});
