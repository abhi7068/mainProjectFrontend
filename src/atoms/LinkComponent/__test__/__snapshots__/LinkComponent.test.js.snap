// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    style="font-family: Roboto, sans-serif; font-weight: 500; font-size: 14px; margin: 0px; white-space: nowrap;"
  />
</DocumentFragment>
`;
